lib/fileHandler/index.js

```diff
@@ -47,19 +47,57 @@ const DEFAULT_UPLOAD_OPTIONS = {
 	queueSize: 4 //Ensures at least up to 1/4 of the RAM is utilised
 };
 
-const REQUIRED_FILE_PROPS = ['name', 'url', 'mime_type', 'expires']
+const REQUIRED_FILE_PROPS = [ 'name', 'url', 'mime_type', 'expires' ];
+const checkIfStr = (val) => { return typeof val === 'string'; };
```

Comment:
just use lodash's isString and isInteger methods

Reply:
cool ye